### DISCUSSION TITLE
Single host client toward F5 BigIP is failing

### DISCUSSION THREAD

**Discussion by 14r5j** [ParallelSSH/parallel-ssh#375] [General]
Hi,
I'm trying to use Parallel-SSH to connect to F5 BigIP (F5 use CentOS as base OS).

This simple code (example from pypi.org) works perfectly toward a Red Hat server, but it fails toward F5 BigIP:

s = SSHClient(<host>,user=<username>, password=<password>, timeout=5, num_retries=1)
res = s.run_command("uname")
for line in res.stdout:
    print(line)

Here is the error message:
pssh.exceptions.AuthenticationError: ('Authentication error while connecting to %s:%s - %s - retries %s/%s', '<IP address>, 22, AuthenticationError(), 1, 1)

I have already tested Paramiko and that works, but I want to use Parallel-SSH.

Is there a way to debug a single SSH client?

(This has been posted on the F5 Community also)

**Comment by pkittenis** [comment]
Try latest client.
